.github/workflows/deployment-images.yaml

```diff
@@ -42,10 +42,14 @@ on:
         required: true
       GHCR_USER_PAT:
         required: true
+      RENT_CORP_GHCR_USER_NAME:
+        required: true
+      RENT_CORP_GHCR_USER_PAT:
+        required: true
```

Comment:
This won't matter to most consumers since this workflow is usually called with `secrets: inherit` rather than with any overridden secrets.